xbmc/guilib/GUIIncludes.h

```diff
@@ -47,13 +47,13 @@ class CGUIIncludes
   void Clear();
 
   /*!
-   \brief Load all include components (defaults, constants, variables, expressions and includes)
-    from the given \code{file}.
-   
-   \param file the file to load
-   \return true if the file was loaded otherwise false
-   */
-  bool Load(const std::string &file);
+  \brief Load all include components (defaults, constants, variables, expressions and includes)
+  from the main entrypoint \code{file}. Flattens nested expressions and expressions in variable
+  conditions after loading all other included files.
+
+  \param file the file to load
+  */
+  void LoadEntrypoint(const std::string &file);
```

Comment:
Then we should name the current `Load` method `Load_Internal` and the `LoadEntrypoint` -> `Load` because we use this wording for most of these methods in our guilib. Sorry for my late response.

BTW could you also add some doxy method docs to the flatten methods to make it clear what they do.